nixos/modules/services/monitoring/prometheus/alertmanager.nix

```diff
@@ -127,6 +127,18 @@ in {
           Extra commandline options when launching the Alertmanager.
         '';
       };
+
+      environmentFile = mkOption {
+        type = types.nullOr types.path;
```

Comment:
Are there concerns around this being added to the store?

Reply:
I checked, it won't be added to the store even when set with `environmentFile = ./local/file` due to how `systemd.services.*.serviceConfig` is implemented.